linters/bandit/plugin.yaml

```diff
@@ -11,13 +11,14 @@ lint:
         - name: lint
           # Custom parser type defined in the trunk cli to handle bandit's JSON output.
           output: bandit
-          run: bandit --exit-zero --format json ${target}
+          run: bandit --exit-zero --format json --output ${tmpfile} ${target}
           success_codes: [0]
+          read_output_from: tmp_file
           batch: true
           cache_results: true
           is_security: true
       issue_url_format: https://pypi.org/project/bandit/
-      known_good_version: 1.7.3
+      known_good_version: 1.7.5
```

Comment:
Do you mind reverting this line OR renaming the snapshot to 1.7.5? The tests should run fine either way, but it's more for repo cleanliness and nightly transparency.

Reply:
```suggestion
      known_good_version: 1.7.3
```